### PR TITLE
Jills Pull request

### PR DESCRIPTION
**Pull request by jillanecook**
Here is my pull request

### CODE REVIEW

File: hallllllo!.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+hallllllo!
```
Comment (Ross-Hunter) [inline hallllllo!.txt]: That is not how you spell "hello"

Comment (psollars) [reply hallllllo!.txt]: I agree, but I'll allow it in this case.

**Comment by Ross-Hunter** [comment]
This looks great!

File: hallllllo!.txt (hunk 1 above)
Comment (jillanecook) [reply hallllllo!.txt]: Won't let it happen again.

**Pull request merged**